### DISCUSSION TITLE
请问一下，可以像Acrobat那样嵌入签名吗？

### DISCUSSION THREAD

**Discussion by songyuc** [wmjordan/PDFPatcher#168] [问答]
尊敬的开发者：

您好！请问一下，[PDFPatcher](https://github.com/wmjordan/PDFPatcher)可以像Acrobat那样嵌入签名吗？就是用手机拍摄一个签名的图片，然后嵌入到PDF页面中去。

期待您的解答！

**Comment by wmjordan** [comment]
你好。目前没有这样的功能。PDF补丁丁是个人开发的小软件，不是大而全的工具。
